aten/src/THNN/generic/VolumetricMaxUnpooling.c

```diff
@@ -63,49 +63,36 @@ static void THNN_(VolumetricMaxUnpooling_updateOutput_frame)(
           int iH,
           int oT,
           int oW,
-          int oH,
-          int dT,
-          int dW,
-          int dH,
-          int pT,
-          int pW,
-          int pH)
+          int oH)
 {
   int k;
   int has_error = 0;
   THIndex_t error_index;
 #pragma omp parallel for private(k)
   for (k = 0; k < nslices; k++)
   {
-    int ti, i, j, maxz, maxy, maxx;
+    real *output_p_k = output_p + k * oT * oH * oW;
+    real *input_p_k = input_p + k * iT * iH * iW;
+    THIndex_t *ind_p_k = ind_p + k * iT * iH * iW;
+
+    int ti, i, j;
+    THIndex_t maxp;
     for (ti = 0; ti < iT; ti++)
```

Comment:
Nit: this is probably nicer as (t, h, w) instead of (ti, i, j).
Same for the usage in maxpool3d.